clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -4833,6 +4879,92 @@ public void testDescribeFeaturesFailure() {
         }
     }
 
+    @Test
+    public void testDescribeMetadataQuorumSuccess() throws Exception {
+        try (final AdminClientUnitTestEnv env = mockClientEnv()) {
+            env.kafkaClient().setNodeApiVersions(NodeApiVersions.create(ApiKeys.DESCRIBE_QUORUM.id,
```

Comment:
nit: You could use `NodeApiVersions.create()`.

Reply:
Not sure about this comment. I think I would need to add the `DESCRIBE_QUORUM` API to the list of known APIs even if we used the default constructor, right? If I do not do that, the test encounters an error where it claims to not know about the `DESCRIBE_QUORUM` API.

Reply:
We can't use `NodeApiVersions.create()` here due to a bug here, which I'm trying to fix here: https://github.com/apache/kafka/pull/11784